src/lightning_app/cli/lightning_cli_delete.py

```diff
@@ -40,6 +45,166 @@ def delete_cluster(cluster: str, force: bool = False, do_async: bool = False) ->
     cluster_manager.delete(cluster_id=cluster, force=force, do_async=do_async)
 
 
+def _cli_delete_app_find_cluster(app_name: str, cluster_id: str) -> str:
+    console = Console()
+    cluster_manager = AWSClusterManager()
+
+    default_cluster, valid_clusters = None, []
+    for cluster in cluster_manager.list_clusters():
+        valid_clusters.append(cluster.id)
+        if cluster.spec.cluster_type == V1ClusterType.GLOBAL and default_cluster is None:
+            default_cluster = cluster.id
+
+    # when no cluster-id is passed in, use the default (Lightning Cloud) cluster
+    if cluster_id is None:
+        cluster_id = default_cluster
+
+    if cluster_id not in valid_clusters:
+        console.print(f'[b][yellow]You don\'t have access to cluster "{cluster_id}"[/yellow][/b]')
+        if len(valid_clusters) == 1:
```

Comment:
if we are planning to have multiple "global clusters" (as per my understanding Karolis was planning to do this sooner than later), this would break